google/resource_compute_instance_group_manager.go

```diff
@@ -38,9 +38,10 @@ func resourceComputeInstanceGroupManager() *schema.Resource {
 			},
 
 			"version": &schema.Schema{
-				Type:     schema.TypeList,
-				Optional: true,
-				Computed: true,
+				Type:       schema.TypeList,
+				Optional:   true,
+				Computed:   true,
+				Deprecated: "Use the instance_group_manager resource in the google-beta provider instead. See https://terraform.io/docs/providers/google/provider-versions.html for more details.",
```

Comment:
What do you think about prefixing these fields with a message that the field is beta? "This field is beta, use the instance_group_manager resource..."

Reading the message as it stands suggests that the resource itself is beta to me.